artemis-cli/src/test/resources/toUpgradeTest/etc/artemis.profile

```diff
@@ -0,0 +1,63 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+ARTEMIS_HOME='/Users/someUser/Downloads/upgrade/apache-artemis-2.25.0'
+ARTEMIS_INSTANCE='/Users/someUser/Downloads/toUpgradeTest'
+ARTEMIS_DATA_DIR='/Users/someUser/Downloads/toUpgradeTest/data'
+ARTEMIS_ETC_DIR='/Users/someUser/Downloads/toUpgradeTest/etc'
+ARTEMIS_OOME_DUMP='/Users/someUser/Downloads/toUpgradeTest/log/oom_dump.hprof'
```

Comment:
these will be changed on ./artemis upgrade --updateInstance now


I would need access to the former version to create an instance in the right place.


this will be just a basic test, and we will need to do actual manual test for the upgrade.